spec/wizards/schools/register_ect_wizard/email_address_step_spec.rb

```diff
@@ -62,8 +63,18 @@
   end
 
   describe '#previous_step' do
-    it 'returns the previous step' do
-      expect(subject.previous_step).to eq(:review_ect_details)
+    context 'with ECTAtSchoolPeriods' do
```

Comment:
```suggestion
    context 'when the teacher had been registered before' do
```

Reply:
...because at this level we only care if the teacher was registered before or not (no matter the way `ect` determines it. By checking ECTAtSchoolPeriods or any other way)

Reply:
Excellent point - updated the contexts to reflect behaviour over implementation 👍🏽